sources/models/shared-defaults/oci-hooks.toml

```diff
@@ -1,13 +1,13 @@
 [settings.oci-hooks]
 log4j-hotpatch-enabled = false
 
-[metadata.settings.oci-hooks]
-affected-services = ["oci-hooks"]
+[metadata.settings.oci-hooks.log4j-hotpatch-enabled]
+affected-services = ["log4j-hotpatch-enabled"]
 
-[services.oci-hooks]
-configuration-files = ["oci-hooks"]
-restart-commands = []
+[services.log4j-hotpatch-enabled]
+configuration-files = ["log4j-hotpatch-enabled-log-message"]
+restart-commands = ["systemctl restart deprecation-warning@log4j-hotpatch-enabled.timer"]
 
-[configuration-files.oci-hooks]
-path = "/etc/shimpei/shimpei-hooks.json"
-template-path = "/usr/share/templates/oci-default-hooks-json"
+[configuration-files.log4j-hotpatch-enabled-log-message]
+path = "/etc/deprecated-settings/log4j-hotpatch-enabled"
+template-path = "/usr/share/templates/log4j-hotpatch-enabled"
```

Comment:
These changes look good to me since they're isolated to the specific `log4j-hotpatch-enabled` setting.